.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+#### What does the PR do?
```

Comment:
Thoughts on:
1. One template, adding an "External Contributor" section with a reminder of CLA requirement
or
2. Two templates: Internal and External Contributor? External one having CLA requirement as well has possibly fewer section like no Pipeline ID?

Reply:
Aiming to reduce the overhead of external PRs having the extra round trip of "Hey, did you sign the CLA?"

Reply:
I think two templates make sense - as it also would could have a section on pre-commit / code ql